config/static_provider.go

```diff
@@ -20,13 +20,7 @@
 
 package config
 
-import (
-	"fmt"
-	"sync"
-)
-
 type staticProvider struct {
-	sync.RWMutex
```

Comment:
Yes, I agree the lock is useless. Just pointing out that StaticProvider wasn't "race-free" even with the lock.